nxt/test/test_resolve.py

```diff
@@ -0,0 +1,26 @@
+# Builtin
+import unittest
+
+# Internal
+from nxt.session import Session
+from nxt.test import get_test_file_path
+from nxt import DATA_STATE
+
+class TestResolve(unittest.TestCase):
+    def test_multiline_code_resolve(self):
+        """When an attr has "real" newline and is subsituted into code,
+        resolve that into multiple code lines
+        """
+        test_graph = get_test_file_path("ResolveNewlines.nxt")
```

Comment:
What is the purpose of this new `get_test_file_path` function?
In all the other tests we _just_ do this:
```suggestion
        test_dir = os.path.dirname(__file__)
        test_graph = os.path.join(test_dir, "ResolveNewlines.nxt")
```

Reply:
Was reading about reliability of `__file__` and this function is more resiliant.

When writing the unit test it was something I wished I didn't have to remember or rewrite again.

I think in general there are some good helpers that can make most unit tests easy to write, I expect to want more, not less.

Reply:
Ah okay, well in that case go for it.
Can you link any good resources on why `__file__` isn't reliable?

Reply:
Had to reverse engineer my search patterns from last night:
https://stackoverflow.com/a/6209894
Turns out the only case is when `execfile` is used, so practially, not meaningfully different.

My other reasons were more important to me.